src/Plugin/EntityBrowser/Widget/View.php

```diff
@@ -209,4 +210,41 @@ public function submit(array &$element, array &$form, FormStateInterface $form_s
     $this->selectEntities($entities, $form_state);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $options = [];
+    // Get only those enabled Views that have entity_browser displays.
+    $displays = Views::getApplicableViews('entity_browser_display');
+    foreach ($displays as $display) {
+      list($view_id, $display_id) = $display;
+      $view = $this->entityManager->getStorage('view')->load($view_id);
+      $options[$view_id . '.' . $display_id] = $this->t('@view : @display', array('@view' => $view->label(), '@display' => $view->get('display')[$display_id]['display_title']));
+    }
```

Comment:
How, when and where should we check that? Y'know, if Views is not a hard dependency, then somebody might set up an entity_browser with a view widget, then uninstall Views – and there might be even more shady cases hidden somewhere. Should we check for Views' availability every time it might be needed? Or… should we add a new property to \Drupal\entity_browser\Annotation\EntityBrowserWidget as an array of dependencies, so every widget could inform Entity Browser about its own dependencies? (Tho I don't yet know how to add that property, nor how, when and where should Entity Browser check a plugin's dependencies.)

And one more question: should this all be handled in this issue/PR, or should we just commit this and handle the dependencies in a different issue?